src/core/server/base.js

```diff
@@ -199,6 +199,16 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
       webpackIsomorphicTools.refresh();
     }
 
+    const cacheAllResponsesFor = config.get('cacheAllResponsesFor');
+    if (cacheAllResponsesFor) {
+      log.warn(oneLine`Sending a Cache-Control header so that the client caches
+        all requests for ${cacheAllResponsesFor} seconds`);
+      res.set('Cache-Control', `public, max-age=${cacheAllResponsesFor}`);
+    }
+
+    // Vary the cache on Do Not Track headers.
+    res.vary('DNT');
```

Comment:
Usually when I see the DNT header written in lowercase, but I presume this doesn't really matter.